### ISSUE TITLE
Worked !  But the question is does it hides my ip ?

### ISSUE THREAD

**Issue by arham56** [BOT-CODER/SniperMan#11]
I tried that and it really worked but the question is that CAN IT HIDE MY IP ????

**Comment by Kip167** [comment]
no, this script will send a couple bytes from your network to the "victim" network, through a direct connection so your ip will be public, granted, the victim probably wont even notice their getting DOSed since this program basically has no power
